src/CMakeLists.txt

```diff
@@ -136,46 +148,18 @@ IF (OpenMVG_BUILD_OPENGL_EXAMPLES)
   SET_PROPERTY(TARGET glfw PROPERTY FOLDER OpenMVG/3rdParty/glfw)
 ENDIF (OpenMVG_BUILD_OPENGL_EXAMPLES)
 
-#===============================
-#---- LINEAR PROGRAMMING SOLVER
-#===============================
-
 #- Mosek (linear programming interface)
 #===============================================================================
 FIND_PACKAGE(Mosek)
 IF(MOSEK_FOUND)
   ADD_DEFINITIONS(-DOPENMVG_HAVE_MOSEK)
-  SET(LP_INCLUDE_DIRS 
+  SET(LP_INCLUDE_DIRS
     ${MOSEK_INCLUDE}
     ${CMAKE_CURRENT_SOURCE_DIR}/dependencies/osi_clp/Osi/src/OsiMsk/
   )
   INCLUDE_DIRECTORIES(${LP_INCLUDE_DIRS})
 ENDIF(MOSEK_FOUND)
 
-#- osi_clp (linear programming interface)
-# ==============================================================================
-if(NOT EXISTS ${PROJECT_SOURCE_DIR}/dependencies/osi_clp/Clp)
```

Comment:
I would love to keep this submodule check.
It helps beginner to ensure they have a valid OpenMVG local repository that is ready for compilation.
What about just change the submodule that is tested, since CLP can now be external...
```
# ==============================================================================
# Check that submodule have been initialized and updated
# ==============================================================================
if(NOT EXISTS ${PROJECT_SOURCE_DIR}/dependencies/cereal/include)
  message(FATAL_ERROR
    "\n submodule(s) are missing, please update your repository:\n"
    "  > git submodule update -i\n")
endif()
```